packages/salesforcedx-vscode-core/test/vscode-integration/commands/util/postCheckers.test.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {
+  ContinueResponse,
+  DirFileNameSelection
+} from '@salesforce/salesforcedx-utils-vscode/out/src/types';
+import { expect } from 'chai';
+import { sandbox, SinonStub } from 'sinon';
+import { Uri, workspace } from 'vscode';
+import { FilePathExistsChecker } from '../../../../src/commands/util';
+import { GlobStrategy } from '../../../../src/commands/util';
+import { nls } from '../../../../src/messages';
+import { notificationService } from '../../../../src/notifications';
+
+const env = sandbox.create();
+
+describe('Postcondition Checkers', () => {
+  describe('FilePathExistsChecker', () => {
```

Comment:
New tests for FilePathExistsChecker. It's now being tested with the glob strategies and independently of the commands that utilize it.